docs/manual/docs/administrator-guide/managing-users-and-groups/index.md

```diff
@@ -76,3 +76,126 @@ Rights associated with the roles are illustrated in detail in the list below:
+    | Delete| | | :material-check-circle: | :material-check-circle: |           DEL           | :material-check-circle: |
+    | Cancel working copy| | | :material-check-circle: | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Privileges| | | :material-check-circle: | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Transfer Ownership| | | | |                         | :material-check-circle: |
+    | Manage Record / Unpublish| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Publish| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / submit for review| | | :material-check-circle: | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / directly approve metadata| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / approve metadata | | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / reject approval submision| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / cancel approval submission| | | :material-check-circle: | |                         | :material-check-circle: |
+    | Manage Record / DOI Creation request| | | :material-check-circle: | :material-check-circle: |           UI            | :material-check-circle: |
+    | Manage Record / Duplicate| | | :material-check-circle: | :material-check-circle: |           UI            | :material-check-circle: |
+    | Manage Record / Create a child| | | :material-check-circle: | :material-check-circle: |           UI            | :material-check-circle: |
+    | Download record / Permalink| :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Download record / Export (ZIP)| :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Download record / Export (PDF)| :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Download record / Export (XML)| :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Download record / EXport (RDF)| :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+
+=== "Editor"
```

Comment:
Similar comment as previous ones, `UserAdmin` should not have access to the editor board.